rusers.h is unused and not available for uClibc-ng / libtirpc

Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>

diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_auth_destroy/rpc_auth_destroy.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_auth_destroy/rpc_auth_destroy.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_auth_destroy/rpc_auth_destroy.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_auth_destroy/rpc_auth_destroy.c	2016-03-05 02:16:56.274105305 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authnone_create/rpc_authnone_create.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authnone_create/rpc_authnone_create.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authnone_create/rpc_authnone_create.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authnone_create/rpc_authnone_create.c	2016-03-05 02:16:32.641196984 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create/rpc_authunix_create.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create/rpc_authunix_create.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create/rpc_authunix_create.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create/rpc_authunix_create.c	2016-03-05 02:21:09.831850681 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create_default/rpc_authunix_create_default.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create_default/rpc_authunix_create_default.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create_default/rpc_authunix_create_default.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_auth_authunix_create_default/rpc_authunix_create_default.c	2016-03-05 02:18:57.370759601 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create.c	2016-03-05 02:23:32.113319210 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_limits.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_limits.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_limits.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_limits.c	2016-03-05 02:23:39.221592413 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_stress.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_stress.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_stress.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clnttcp_create/rpc_clnttcp_create_stress.c	2016-03-05 02:23:48.565951560 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate.c	2016-03-05 02:23:56.706264429 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate_limits.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate_limits.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate_limits.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_bufcreate/rpc_clntudp_bufcreate_limits.c	2016-03-05 02:24:03.818537786 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create.c	2016-03-05 02:24:10.518795308 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
diff -Nur ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create_stress.c ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create_stress.c
--- ltp-full-20160126.orig/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create_stress.c	2016-01-26 13:35:25.000000000 +0100
+++ ltp-full-20160126/testcases/network/rpc/rpc-tirpc/tests_pack/rpc_suite/rpc/rpc_createdestroy_clntudp_create/rpc_clntudp_create_stress.c	2016-03-05 02:24:20.095163371 +0100
@@ -30,7 +30,6 @@
 #include <time.h>
 #include <rpc/rpc.h>
 #include <sys/socket.h>
-#include <rpcsvc/rusers.h>
 #include <utmp.h>
 #include <sys/time.h>
 #include <netdb.h>
